src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -232,6 +233,25 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
     return js_devices;
 }
 
+Napi::Value CoreWrap::get_versions(const Napi::CallbackInfo& info) {
+    const auto& devices_map = _core.get_versions(info[0].As<Napi::String>().Utf8Value());
+    Napi::Array js_devices = Napi::Array::New(info.Env(), devices_map.size());
```

Comment:
Why array should have size = passed string length?

Reply:
`_core.get_versions` returns a map containing the device and the deviceVersion. Thus the length of js_devices array should be the same as that of devices_map.
I have referred to this doc for the same. 
```std::map<std::string, ov::Version> ov::Core::get_versions(const std::string &device_name) const
Returns device plugins version information. Device name can be complex and identify multiple devices at once like HETERO:CPU,GPU; in this case, std::map contains multiple entries, each per device.

Parameters:
device_name – Device name to identify a plugin.

Returns:
A vector of versions.
```

Reply:
My bad, I wrongly interpreted map type.